eventdata/runners/fieldstats_runner.py

```diff
@@ -29,8 +29,16 @@ def fieldstats(es, params):
     
     if 'fieldname' not in params:
         params['fieldname'] = '@timestamp'
+    
+    ignore_frozen = True
```

Comment:
I think `ignore_frozen` should be provided in every case. Then you can just write:

```python
ignore_frozen = params['ignore_frozen']
```

and remove all the conditional logic below.